### DISCUSSION TITLE
0B42A55F-59A0-412A-A3CD-41BE9898DC0A

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#188] [General]
# 0B42A55F-59A0-412A-A3CD-41BE9898DC0A

Kepler-450, also known as KOI-279, is a binary star system (consisting of a main-sequence star of spectral type F7 and a main-sequence star of spectral type K) located 1571.079 light-years from Earth, and is known to have 3 planets, two Neptune-like objects and a super-Earth, all of which have orbital periods of no more than 30 days

https://celestia.mobi/addon?item=0B42A55F-59A0-412A-A3CD-41BE9898DC0A

**Comment by exocritic5457** [comment]
....."interesting" colors.

**Reply by Galaxy-Leaper1** [reply]
First of all, please don't make sarcastic comments, and secondly, my creative level is really limited because my equipment doesn't support making more complex textures, so most of them are neon colored, for exoplanets, it is difficult for us to observe their surface features, the textures of these add-ons are just fictional, you can make them according to your own ideas, as long as it is reasonable
